plugin/plugintest/hooks.go

```diff
@@ -199,6 +199,16 @@ func (_m *Hooks) OnPluginClusterEvent(c *plugin.Context, ev model.PluginClusterE
 	_m.Called(c, ev)
 }
 
+// OnWebSocketConnect provides a mock function with given fields: webConnID, userID
+func (_m *Hooks) OnWebSocketConnect(webConnID string, userID string) {
```

Comment:
Trying to make sure all mocks were up to date, I stumbled upon these changes. We cna piggyback them in this PR, or revert them and add a different PR to update mocks.